catalog_tools/analysis/estimate_beta.py

```diff
@@ -53,7 +53,41 @@ def estimate_beta_utsu(magnitudes: np.ndarray, mc: float, delta_m: float = 0
     return beta
 
 
-def estimate_beta_elst(magnitudes: np.ndarray) -> float:
+def differences(magnitudes: np.ndarray) -> np.ndarray:
+    """returns all the differences between the magnitudes.
+
+    Args:
+        magnitudes: vector of magnitudes differences, sorted in time (first
+                    entry is the earliest earthquake)
+
+    Returns: array of positive differences of the input
```

Comment:
this description is different from the one above, and I think the one above is the correct one.